_posts/en/2018-06-11-setting-users-up for-success-with-content-design.html

```diff
@@ -0,0 +1,102 @@
+---
+title: "Setting up users for success with content design"
+description:
+author: Zak Kain, Content Designer
+date: 2018-06-11 09:00:00 -0400
```

Comment:
Heads up comms. is going to probably make you change these dates, so I'd suggest changing them to the 12th if you're going to post this today :D